src/neuvector/chart/templates/uds-package.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: uds.dev/v1alpha1
+kind: Package
```

Comment:
This is neat! Wanted to call out the tradeoff between:
1. Having a YAML schema that the K8s community understands (ie. NetworkPolicy), but being way more verbose
2. Having a custom YAML schema generated by the fluent client that's more concise at the cost of readability (if you're not sure how the fluent client works)

Reply:
I personally find the K8s NetPol to be one of the worst specs in core with how easy it is to misuse the matchers. Since we'll have many UDS Package CRs, I think the cost benefit is clear vs manually maintaining all these NetPols as Big Bang does today. This includes things like wiring API Server endpoints properly & auto-wiring Ingress policies for VirtualServices.  The current way (which requires maintaining a number of copy/paste policies in addition to per-app ones) is also almost 4 times as much YAML for the same config, only spread through helm chart templates and flux / helm post rendering.

![Screenshot 2024-01-04 at 11 04 36 PM](https://github.com/defenseunicorns/uds-core/assets/882485/7404dc0d-823e-4b23-8993-fd1a9341387a)